app/build.gradle

```diff
@@ -41,6 +41,14 @@ dependencies {
     implementation "androidx.appcompat:appcompat:${appCompatVersion}"
     implementation "com.google.android.material:material:${designSupportVersion}"
 
+    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
```

Comment:
i think we can split the commit into several: 1. adaptor and basic verification that it works 2. home view 3. log out view